operator/redisfailover/service/generator.go

```diff
@@ -254,7 +254,7 @@ func generateRedisStatefulSet(rf *redisfailoverv1.RedisFailover, labels map[stri
 			ServiceName: name,
 			Replicas:    &rf.Spec.Redis.Replicas,
 			UpdateStrategy: appsv1.StatefulSetUpdateStrategy{
-				Type: v1.OnDeleteStatefulSetStrategyType,
+				Type: v1.RollingUpdateStatefulSetStrategyType,
```

Comment:
This is the only real change, plus the image repo in the Makefile. The rest is just new path needed for building